generators/app/templates/src/constants/README.md

```diff
@@ -0,0 +1,3 @@
+# Constants
```

Comment:
What's this?

Reply:
A placeholder to commit this empty folder